docs/AAD-OAuth.md

```diff
@@ -322,18 +325,157 @@ curl -v -H "Authorization: Bearer $acr_access_token" https://$registry/v2/_catal
 ```
 Note that `curl` by default does the request as a `GET` unless you specify a different verb with the `-X` modifier.
 
-This should result in a status 200 OK, and a body with a JSON payload listing the repositories held in this registry:
+This should result in a status 200 OK, and a body with a JSON payload listing the repositories held in this registry, using `...` to shorten it for illustrative purposes:
+
+```json
+{"repositories":["alpine","contoso-marketing","hello-world","node",...]}
+```
+
+#### Pagination
+
+To retrieve paginated catalog results, add an `n` parameter to limit the number or results. We take `n=2` as example:
+
+```bash
+export registry="contosoregistry.azurecr.io"
+export acr_access_token="eyJ...xcg"
+export limit=2
+curl -v -H "Authorization: Bearer $acr_access_token" https://$registry/v2/_catalog?n=$limit
+```
+
+This should result in a status 200 OK, and a body with a JSON payload listing the first `n` repositories held in this registry. If there are more results, a `Link` header containing the request URL for the next result block is also returned.  If the entire result set has been received, the `Link` header will not be returned.
+
+In this case, the first 2 repositories are returned, and there are more entries in the result set. The response would look like:
+
+```bash
+< HTTP/1.1 200 OK
+...
+Content-Type: application/json
+Link: </v2/_catalog?last=contoso-marketing&n=2&orderby=>; rel="next"
+{"repositories": ["alpine","contoso-marketing"]}
+```
+
+To get the next result block, issue the request using the `/v2/_catalog?last=contoso-marketing&n=2&orderby=` URL encoded in the `Link` header. Here's how the call would look like:
```

Comment:
nit: General English writing: no abbr. in formal writting: `Here's` -> `Here is`.